### ISSUE TITLE
[Bug]: Compiled with problems eslint & DEP_WEBPACK_COMPILATION_ASSETS

### ISSUE THREAD

**Issue by nklowns** [wallabyjs/console-ninja#87]
### What happened?

I am getting these errors when I start console ninja IN MY vue2 Project

package.json
```json
"devDependencies": {
    "@babel/eslint-parser": "^7.12.16",
    "@vue/cli-plugin-eslint": "~5.0.0",
    "eslint": "^7.32.0",
    "eslint-config-prettier": "^8.3.0",
    "eslint-plugin-prettier": "^4.0.0",
    "eslint-plugin-vue": "^8.0.3",
    "prettier": "^2.4.1",
},
"eslintConfig": {
    "root": true,
    "env": {
      "node": true,
      "jest": true
    },
    "extends": [
      "plugin:vue/essential",
      "eslint:recommended",
      "plugin:prettier/recommended"
    ],
    "parserOptions": {
      "parser": "@babel/eslint-parser"
    },
    "rules": {},
    "overrides": [
      {
        "files": [
          "**/__tests__/*.{j,t}s?(x)",
          "**/tests/unit/**/*.spec.{j,t}s?(x)"
        ]
      }
    ]
},
"prettier": {
    "singleQuote": true,
    "semi": false,
    "trailingComma": "es5",
    "arrowParens": "avoid",
    "tabWidth"
}
```

### Version

0.0.69

### Steps to reproduce (or sample repo)

As soon as I start console ninja, I am getting these compiled with problem errors.
When I pause console ninja the issue goes away ( most of the time )

Using this eslint, prettier plugins in my project

```cmd
> yarn serve
yarn run v1.22.19
$ vue-cli-service serve
 INFO  Starting development server...
[95%] emitting (emit CopyPlugin)
(node:16184) [DEP_WEBPACK_COMPILATION_ASSETS] DeprecationWarning: Compilation.assets will be frozen in future, all modifications are deprecated.
BREAKING CHANGE: No more changes should happen to Compilation.assets after sealing the Compilation.
        Do changes to assets earlier, e. g. in Compilation.hooks.processAssets.
        Make sure to select an appropriate stage from Compilation.PROCESS_ASSETS_STAGE_*.


 ERROR  Failed to compile with 1 error                                                                                                                                                                                                                    11:51:47

[eslint] 
C:\develop\src\App.vue
  28:9  error  Delete `··········⏎··················⏎···········································⏎⏎························⏎··················⏎·······························⏎·········⏎·······················⏎·····⏎··············································⏎························································⏎················································································⏎···················⏎···············⏎··························⏎⏎·························⏎················································⏎····························································⏎····················⏎⏎·································⏎·············⏎··········⏎···········⏎⏎········`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.


You may use special comments to disable some warnings.
Use // eslint-disable-next-line to ignore the next line.
Use /* eslint-disable */ to ignore all warnings in a file.
ERROR in [eslint] 
C:\develop\src\App.vue
  28:9  error  Delete `··········⏎··················⏎···········································⏎⏎························⏎··················⏎·······························⏎·········⏎·······················⏎·····⏎··············································⏎························································⏎················································································⏎···················⏎···············⏎··························⏎⏎·························⏎················································⏎····························································⏎····················⏎⏎·································⏎·············⏎··········⏎···········⏎⏎········`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.


webpack compiled with 1 error
```

### Log output

```shell
info extension stopping host
info extension host process output: port:[53836]
info host ------------ starting ------------
info extension 
info host client connected: extension { address: '::', family: 'IPv6', port: 53836 }
info host worker Starting Host Worker...
info host starting host: {"extensionFolder":"c:\\Users\\cloud\\.vscode\\extensions\\wallabyjs.console-ninja-0.0.69","runtimeHookFile":"c:\\Users\\cloud\\.vscode\\extensions\\wallabyjs.console-ninja-0.0.69\\out\\runtimeHook\\tracer.js","globalErrorHandlerFile":"c:\\Users\\cloud\\.vscode\\extensions\\wallabyjs.console-ninja-0.0.69\\out\\runtimeHook\\errorHandler.js","filesToInstrument":["C:\\develop\\comeia\\teamculture-ui-performance\\src\\App.vue"],"debug":false,"config":{"dateTimeFormat":"hh:mm:ss.SSS","openLinksInEditor":true,"maxLogViewerEntries":15000,"autoClearLogViewerOnFileChanges":true,"outputMode":"Beside File","captureFunctions":false,"toolsToAutoPatch":["vite","jest","webpack","next.js","cypress","http-server","serve","nuxt","remix","hydrogen"]},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","GENESIS-NKS","192.168.18.35"],"host":"127.0.0.1"}
info host starting tools discovery and integration
info host looking for tools to patch
info host found 11 tool file(s) to patch
info host patching C:\develop\node_modules\webpack\lib\index.js
info host patching C:\develop\node_modules\webpack\lib\webpack.js
info host patching C:\develop\node_modules\style-loader\node_modules\webpack\lib\webpack.js
info host patching C:\develop\node_modules\webpack-dev-middleware\node_modules\webpack\lib\webpack.js
info host patching C:\develop\node_modules\@storybook\builder-webpack4\node_modules\webpack\lib\webpack.js
info host patching C:\develop\node_modules\@storybook\core-common\node_modules\webpack\lib\webpack.js
info host patching C:\develop\node_modules\@storybook\core-server\node_modules\webpack\lib\webpack.js
info host patching C:\develop\node_modules\@storybook\manager-webpack4\node_modules\webpack\lib\webpack.js
info host patching C:\develop\node_modules\jest\bin\jest.js
info host patching C:\develop\node_modules\jest-runner\build\testWorker.js
info host patching C:\develop\node_modules\cypress\lib\cypress.js
info host completed tools discovery and integration
info buildHook-24688 installing build hook for webpack at C:\develop
info buildHook-24688 allowed tools: vite, jest, webpack, next.js, cypress, http-server, serve, nuxt, remix, hydrogen
info host client connected: buildHook { address: '::', family: 'IPv6', port: 53836 }
info buildHook-24688 running tools: webpack
info buildHook-24688 installing fs interceptor
info buildHook-24688 file processed (stat): 0 log points, error handler: true
info buildHook-24688 build hook already installed
info buildHook-24688 file processed (stat): 1 log points, error handler: false
info buildHook-24688 allowed tools: vite, jest, webpack, next.js, cypress, http-server, serve, nuxt, remix, hydrogen
info buildHook-24688 running tools: webpack
```

**Comment by nklowns** [comment]
Similar to the issue https://github.com/wallabyjs/console-ninja/issues/85
but the project I'm running has different lint rules like prettier.

**Comment by NikGovorov** [comment]
Thanks for reporting the issue. It is fixed and the fix is published in v0.0.70.
